src-tauri/tauri.conf.json

```diff
@@ -12,7 +12,16 @@
   },
   "tauri": {
     "allowlist": {
-      "all": true
+      "all": true,
+      "path": {
+        "all": true
+      },
+      "fs": {
+        "all": true,
+        "scope": [
+          "**"
+        ]
+      }
```

Comment:
Required so the tauri fs API has access to read files

Reply:
Really? I thought `"all": true` enables everything.

I intend to reduce the allowed scopes before release, but I believe that with the current settings everything is allowed.

Reply:
Tauri was raising permissions errors on file reads without adding this.  Maybe "all" only applies to top-level permissions within the scope?

Reply:
That's odd, but I suppose you're right.

Reply:
Is it sufficient to specify the scope?